### DISCUSSION TITLE
Including VS Code Extensions in this repository?

### DISCUSSION THREAD

**Discussion by anton-roos** [carbon-language/carbon-lang#1679] [Toolchain]
Hi, @jskf created a VS Code extension:

https://github.com/JSKF/Carbon-VSCode-Language-Server
https://github.com/JSKF/CarbonVSCode

Would there be any merit in having this included in the carbon-lang repo and toolchain?

**Comment by geoffromer** [comment]
Yeah, we've generally been pretty open to adding editor/highlighting support to the main repo -- see the `utils` directory.

**Comment by anshiq** [comment]
Hi there, Is there any vs code syntax highlighter present or not, if not I am bulding carbon lang vs code extension, should keep working  on it.
